spec/menu_item_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'menu_item'
+
+describe Menu_item do
+
+
+
+  it 'initialize raises error when missing menu name and price' do
+    expect{described_class.new()}.to raise_error(ArgumentError)
```

Comment:
Style/MethodCallParentheses: Do not use parentheses for method calls with no arguments.